lib/katello/tasks/prepare_content_overrides_for_sca.rake

```diff
@@ -0,0 +1,7 @@
+namespace :katello do
+  desc "Add disabled content overrides for custom products which do not have a subscription attachment"
```

Comment:
Do you think it would be worth adding a bit about why you would need this rake task?